aws/rust-runtime/aws-http/Cargo.toml

```diff
@@ -2,24 +2,13 @@
 name = "aws-http"
 version = "0.0.0-smithy-rs-head"
```

Comment:
I think you need to change this version to some sort of EOL version? we need to be careful **not** to publish this with the next set of unstable crates

Reply:
Good call. Updated.